src/screens/ForgotPassword.js

```diff
@@ -1,105 +1,98 @@
-import { Pressable, StyleSheet, Text, TextInput, View } from 'react-native'
+import { Pressable, StyleSheet, Text, TextInput, View, SafeAreaView, Dimensions } from 'react-native'
 import React, { useState } from 'react'
+import Icon from 'react-native-vector-icons/FontAwesome';
```

Comment:
```suggestion
import Icon from 'react-native-vector-icons/FontAwesome';
```


Projenin Package.js dosyasında indirilmiş olan npm paketleri dışında paketi projeye eklememelisiniz. Burada expo icons kullanılması gerekiyor.